prometeo/mem/ast_analyzer.py

```diff
@@ -22,13 +22,15 @@
     'global@pmt_gemm_tn': [], \
     'global@pmt_gemm_nt': [], \
     'global@pmt_trmm_rlnn': [], \
+    'global@pmt_syrk_ln': [], \
     'global@pmt_gead': [], \
     'global@pmt_potrf': [], \
     'global@pmt_potrsm': [], \
```

Comment:
What is `potrsm`? I guess you wanted to refer to `potrs`, same for `getrs`.

Reply:
same as above for `getrsm`